server/text_generation_server/utils/logits_process.py

```diff
@@ -623,3 +623,112 @@ def filter(self, indices):
             new_fsms.append(self.fsms[i])
         self.fsms = new_fsms
         return self
+
+
+class LogitBiasProcessor:
+    """Process logits with logit biases."""
+
+    def __init__(
+        self, logit_biases: Optional[dict], tokenizer: PreTrainedTokenizerBase
+    ):
+        self.tokenizer = tokenizer
+        self.logit_biases = logit_biases or {}
+
+        # Pre-compute token IDs for each token string
+        self.token_id_mapping = {}
```

Comment:
Where is the pre-computing ?

Reply:
updated along with logit bias processor changes